core/commonMain/src/kotlinx/serialization/internal/PluginGeneratedSerialDescriptor.kt

```diff
@@ -37,12 +37,12 @@ internal open class PluginGeneratedSerialDescriptor(
     private val childSerializers by lazy { generatedSerializer?.childSerializers() ?: emptyArray() }
 
     // Lazy because of JS specific initialization order (#789)
-    private val typeParameterDescriptors: Array<SerialDescriptor> by lazy {
+    internal val typeParameterDescriptors: Array<SerialDescriptor> by lazy {
         generatedSerializer?.typeParametersSerializers()?.map { it.descriptor }.compactArray()
     }
 
     // Can be without synchronization but Native will likely break due to freezing
-    private val _hashCode: Int by lazy { hashCodeImpl(typeParameterDescriptors) }
+    internal val _hashCode: Int by lazy { hashCodeImpl(typeParameterDescriptors) }
```

Comment:
It could be left private and `hashCode()` can be used instead

Reply:
You mean `super.hashCode()`? Yeah, why not